lib/elixlsx/sheet.ex

```diff
@@ -110,6 +114,8 @@ defmodule Elixlsx.Sheet do
   """
   def set_at(sheet, rowidx, colidx, content, opts \\ [])
       when is_number(rowidx) and is_number(colidx) do
+    sheet = maybe_extend(sheet, rowidx, colidx)
+
     cond do
       length(sheet.rows) <= rowidx ->
         # append new rows, call self again with new sheet
```

Comment:
The below code is now mostly handled by maybe_extend / dead, if I'm not mistaken?